interface/ceed-fortran.c

```diff
@@ -105,10 +105,11 @@ void fCeedView(int *ceed, int *err) {
 
 #define fCeedDestroy FORTRAN_NAME(ceeddestroy,CEEDDESTROY)
 void fCeedDestroy(int *ceed, int *err) {
-  if (Ceed_n == 0 || !Ceed_dict[*ceed]) return;
+  if (Ceed_n == 0 || *ceed == FORTRAN_NULL || !Ceed_dict[*ceed]) return;
```

Comment:
Could we simplify to this?
```suggestion
  if (*ceed == FORTRAN_NULL || !Ceed_dict[*ceed]) return;
```

Reply:
I also cut the third condition in my latest commit. I think `*ceed == FORTRAN_NULL` is the only test needed